src/tiled/stampbrush.cpp

```diff
@@ -143,10 +143,7 @@ void StampBrush::mousePressed(QGraphicsSceneMouseEvent *event)
             }
             return;
         } else if (event->button() == Qt::RightButton) {
-            if (event->modifiers() == Qt::NoModifier)
-                beginCapture(false);
-            else if (event->modifiers() & Qt::ShiftModifier)
-                beginCapture(true);
+            beginCapture();
```

Comment:
Here also we'll need to check that the Ctrl modifier isn't used, since in that case we want the ctrl+right-click handling in `AbstractTileTool::mousePressed` to kick in.